### PR TITLE
Improve doc generation

### PR DESCRIPTION
**Pull request by Ellipse0934**
- Changed theme to Documenter.jl Vanilla
- small cleanups

### CODE REVIEW

File: docs/make.jl [hunk 1]
```diff
@@ -31,13 +30,12 @@ makedocs(
...
             "Sky Scraper" => "example/skyscraperBasic.md",
             "Game Of Life" => "example/GameOfLife.md",
-            "Platform 9¾" => "example/platform9_75.md"
+            "Platform 9¾" => "example/platform9_75.md",
             "Whac-a-Block" => "whacABlock.md"
        ],
        "Block Reference" => "blocks.md",
        "API Reference" => "api.md"
-   ],
-   assets = ["assets/custom.css"]
```
Comment (aviks) [inline docs/make.jl:40]: we don't need a custom css any more?

File: docs/src/api.md [hunk 2]
```diff
@@ -6,62 +6,4 @@
 ```@autodocs
 Modules = [PiCraft]
 Order   = [:type, :function]
-```
```
Comment (aviks) [inline docs/src/api.md:9]: autodocs is working now?

File: docs/make.jl (hunk 1 above)
Comment (Ellipse0934) [reply docs/make.jl:40]: I dislike the current look, appears quite dated and non-modern to me. I may touch it up with a bit of minecrafty theme later.

File: docs/src/api.md (hunk 2 above)
Comment (Ellipse0934) [reply docs/src/api.md:9]: yes. It dumps all commands together. I'll tidy it up soon by grouping relevant commands together.

**Comment by Ellipse0934** [comment]
> deploydocs(
    repo = "github.com/aviks/PiCraft.jl.git",

I'm not sure if I can deploy docs. Can you deploy it ? or maybe a very short summary of what I need to do to deploy it.

**Pull request merged**